_posts/2019-04-25-safer-and-cleaner-seas-stm.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: post
+title: Safer and cleaner seas via Sea Traffic Management
+author: arto
+excerpt: Air traffic has been digitalized long ago. Now, the seas are going same route as well. Future holds much safer and cleaner marine traffic for all of us.
+tags:
+- Environment
+- Automation
+- Digitalization
+- Project
+- Distributed
+- Java
+- XML
+---
+**Human errors can be expensive**. When you are making decisions, and acting on them while not fully concentrating, it leaves room for mistakes. A typical human error might be writing your address wrong when ordering an item. Or copying a single digit of bank account number wrong, from a handwritten note. Perhaps you are acting on too little information, or are in a hurry. Those mistakes happen every day to someone. In most cases you can recover from them through some extra work, no harm done.
```

Comment:
"When you are making decisions, and acting on them...." ---> When you are making decisions and acting on them without your full concentration, mistakes are likely to happen.

It leaves room for mistakes kuulostaa siltä, kuin jäisi pelivaraa virheiden suhteen. Ehdottamani sanamuoto kertoisi asian suoremmin.

Reply:
"Or copying a single digit of bank account number wrong, from a handwritten note."

Kannattaa aloittaa virke jollain muulla sanalla kuin "Or". Ei tarvitse pilkkua tässä.

"You might mistype a bank account number when copying from a poorly handwritten note."

Reply:
"Perhaps you are acting on too little information, or are in a hurry. Those mistakes happen every day to someone."

olisiko sujuvampi näin?

"These kinds of every day mistakes happen all the time when acting in a hurry or based on incomplete information."

Reply:
Olisi :) Kiitoksia.